desktop/core/src/desktop/lib/i18n.py

```diff
@@ -59,8 +60,13 @@ def smart_unicode(s, strings_only=False, errors='strict', encoding=None):
   Wrapper around Django's version, while supplying our configured encoding.
   Decode char array to unicode.
```

Comment:
Should probably add a note that this is becoming a 'string' now (no more 'unicode' in py3)